theme/js/locales/en.json

```diff
@@ -109,7 +109,6 @@
   "Data schemas allow to describe data models : what are the fields, how is data represented, what are the possible values. Discover how schemas improve data quality and use cases on ": "Data schemas allow to describe data models : what are the fields, how is data represented, what are the possible values. Discover how schemas improve data quality and use cases on ",
   "Available": "Available",
   "Unavailable": "Unavailable",
-  "Come try out our": "Come try out our {dataset_search}.",
   "new dataset search": "new dataset search",
```

Comment:
Shouldn't `new dataset search` be removed as well?

Reply:
Done !